python/python/tests/test_async_client.py

```diff
@@ -4047,6 +4047,31 @@ async def test_setbit(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.setbit(set_key, 0, 0)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_getbit(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        non_existing_key = get_random_string(10)
+        set_key = get_random_string(10)
+        value = "foobar"
+
+        assert await redis_client.set(key, value) == OK
+        assert await redis_client.getbit(key, 1) == 1
+        assert await redis_client.getbit(key, 1000) == 0
+        assert await redis_client.getbit(non_existing_key, 1) == 0
+
+        assert await redis_client.setbit(key, 5, 0) == 1
```

Comment:
either change this to index 1 (above, it returns 1) or do a `getbit` prior to show that the value was `1` and now you're changing it to `0`.

Reply:
although, honestly, I'm not sure what's the point of this and the next line.

Reply:
yeah this was copied from the java tests but it doesn't seem necessary since we already are have tested regular usage of getbit earlier, I'll remove these 2 lines